docs/scripts_md/get_dicom.mp

```diff
@@ -0,0 +1,45 @@
+# NAME
```

Comment:
name of the doc script was supposed to have extension `.md`. It is `.mp` for some reason.

(Can't believe I noticed that...)

Reply:
Fixed.